packages/hash/api/src/graphql/resolvers/index.ts

```diff
@@ -150,8 +150,7 @@ export const resolvers = {
     executeGithubReadTask: loggedInAndSignedUp(executeGithubReadTask),
   },
 
-  JSONObject: GraphQLJSON,
-  TextToken: GraphQLJSON,
```

Comment:
Keeping `TextToken: JSONObjectResolver` produces the following error:

```
Duplicate schema type name JSONObject {"service":"api","stack":"Error: Duplicate schema type name JSONObject\n    at rewireTypes (/Users/ak/-/projects/hash/github/hash/node_modules/@graphql-tools/utils/cjs/rewire.js:22:19)\n    at mapSchema (/Users/ak/-/projects/hash/github/hash/node_modules/@graphql-tools/utils/cjs/mapSchema.js:13:65)\n    at createNewSchemaWithResolvers (/Users/ak/-/projects/hash/github/hash/node_modules/@graphql-tools/schema/cjs/addResolversToSchema.js:200:36)\n    at addResolversToSchema (/Users/ak/-/projects/hash/github/hash/node_modules/@graphql-tools/schema/cjs/addResolversToSchema.js:89:11)\n    at makeExecutableSchema (/Users/ak/-/projects/hash/github/hash/node_modules/@graphql-tools/schema/cjs/makeExecutableSchema.js:80:65)\n    at createApolloServer (/Users/ak/-/projects/hash/github/hash/packages/hash/api/src/graphql/createApolloServer.ts:47:46)\n    at main (/Users/ak/-/projects/hash/github/hash/packages/hash/api/src/index.ts:194:42)"}
```

It is probably related to `TextToken` in codegen config.

Reply:
@nathggns do you recall what this `TextToken: GraphQLJSON` declaration was for? And what issues removing it might cause?

Reply:
So `TextToken` was introduced by @kachkaev as a scalar in https://github.com/hashintel/dev/pull/331 (internal link) which added a `GraphQLJSON`. It's a scalar so you don't have to specify the individual properties on a text token as they're all necessary. I don't know why the resolver is necessary though. 

@kachkaev you may know?

Reply:
We define `salars` → `TextToken: "@hashintel/hash-shared/graphql/types#TextToken"` in `packages/hash/api/codegen.yml`. This does not create a resolver, but gives us the right typings. I believe that the old version of the GraphQL server was ignoring the fact that `TextToken` scalar is a JSON _and_ also the `TextToken` interface. The new version classifies this as a duplicate.

Reply:
@nathggns We define `salars` → `TextToken: "@hashintel/hash-shared/graphql/types#TextToken"` in `packages/hash/api/codegen.yml`. This does not create a resolver, but gives us the right typings. I believe that the old version of the GraphQL server was ignoring the fact that `TextToken` scalar is a JSON _and_ also the `TextToken` interface. The new version classifies this as a duplicate.

Reply:
Text editing seems to be working fine (it's triggering a huge number of versions when making small changes to a page with many text paragraphs, but this is happening on `main` too – have created [a task](https://app.asana.com/0/1201095311341924/1202583014332039/f))